tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -65,6 +65,202 @@ public function testCreateSite(): void
         $this->cleanupSite($siteId);
     }
 
+    public function testVariables(): void
+    {
+        // create site
+        $site = $this->createSite([
```

Comment:
```suggestion
        $site = $this->createSite([
```

Reply:
Lets update this in in all the changes, there is always comment, and then a call to method of same name. The comment is not nessessary in those scenarios.

We can keep it in places like "get secret variable", when method call doesnt clearly explain if it's a secret or not